CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.4 / 2020-06-09
+
+* [Added] Add Logs Archives endpoints. See [#333](https://github.com/DataDog/datadog-api-client-java/pull/333).
+* [Changed] Add missing values to enums. See [#327](https://github.com/DataDog/datadog-api-client-java/pull/327).
```

Comment:
Can we use `[BREAKING]` for anything with backwards incompatibility (to be consistent with the previous releases)